src/chrome/content/rules/ReadTheDocs.xml

```diff
@@ -20,11 +23,13 @@
 	
 	<test url="http://testing.readthedocs.org/" />
 	<test url="http://blog.readthedocs.com/" />
+	<test url="http://docs.readthedocs.io/" />
```

Comment:
@patryk from memory you'll need 3 urls per wildcard rule.

You could use
`http://readthedocs.io`
`http://www.readthedocs.io`
`http://jupyter.readthedocs.io`

And use those for `.org` and `rtfd.org` too

Popular subdomains can be found on http://www.alexa.com/siteinfo/readthedocs.org ctrl-f "Where do visitors go"